itest/lnd_multi-hop_force_close_test.go

```diff
@@ -328,11 +328,18 @@ func runLocalClaimOutgoingHTLC(ht *lntest.HarnessTest,
 	}
 	ht.SendPaymentAssertInflight(alice, req)
 
-	// Verify that all nodes in the path now have two HTLC's with the
-	// proper parameters.
-	ht.AssertActiveHtlcs(alice, dustPayHash, payHash)
-	ht.AssertActiveHtlcs(bob, dustPayHash, payHash)
-	ht.AssertActiveHtlcs(carol, dustPayHash, payHash)
+	// At this point, all 3 nodes should now have an active channel with
+	// the created HTLC pending on all of them.
+	//
+	// Alice should have two outgoing HTLCs on channel Alice -> Bob.
+	ht.AssertNumActiveHtlcs(alice, 2)
```

Comment:
nit: can we no longer check payment hashes of HTLCs. Can `AssertOutgoingHTLCActive` and
`AssertIncomingHTLCActive` be used to check it?

Reply:
yes! we can either do,
```go
ht.AssertOutgoingHTLCActive(alice, xxx)
ht.AssertIncomingHTLCActive(alice, xxx)
```

or a single
```go
ht.AssertNumActiveHtlcs(alice, 2)
```

because when we reach it here, it's not possible for Alice to have unrelated HTLCs - when there are 2, it must be she has one outgoing and one incoming, and the check-num-approach is preferred as it's cleaner. This becomes more obvious if we replace below `ht.AssertNumActiveHtlcs(bob, 4)`.